Task3/Task3.c

```diff
@@ -0,0 +1,193 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#define BASESIZE 100
+#define LEN 100
+
+typedef struct
+{
+    char name[LEN];
+    char phone[LEN];
+} Entry;
+
+int readAndCountEntriesFromFile(Entry arrayOfEntries[], char* filename)
+{
+    int entriesCounter = 0;
+    FILE* file = fopen(filename, "a+");
+    if (file == NULL)
+    {
+        return -1;
+    }
+    while (!feof(file))
+    {
+        if ((fscanf(file, "%s", arrayOfEntries[entriesCounter].name) != EOF))
+        {
+            fscanf(file, "%s", arrayOfEntries[entriesCounter].phone);
+            entriesCounter++;
+        }
+    }
+    fclose(file);
+    return entriesCounter;
+}
+
+void begin()
```

Comment:
```suggestion
void printHelpPessage()
```
Более информативное имя функции